include/crm/crm.h

```diff
@@ -53,7 +53,14 @@ extern "C" {
  */
 #  define CRM_FEATURE_SET		"3.7.2"
 
+//!@{
+//! \deprecated This will be removed in a future release
+
 #  define EOS		'\0'
```

Comment:
This is also defined in include/portability.h, and is only used at one spot in lib/common/strings.c.  So we could probably get rid of our use entirely and just leave the deprecated macros around for whoever might be using it.